### DISCUSSION TITLE
Image Stitching Error

### DISCUSSION THREAD

**Discussion by rayanelahmadi** [OpenStitching/stitching#220] [Q&A]
Hello,

I get the following error (check attached image) when trying to stitch two images together. It looks like the issue has to do with the camera_adjuster.adjust() function. Additionally, I had a couple of questions in regards to the stitching process as a whole:

1. If I have multiple cameras vertically stacked and each camera is displaced by x inches (all looking straight ahead) and each captures an image, and each image has sufficient overlap, will using AffineStitcher instead of Stitcher make a difference? I've tested both with a set of images and I can't get a clear answer for which one I should be using. 

2. Say I have 3 cameras vertically stacked and each camera is displaced by x inches (all looking straight ahead) and each captures an image, and each image has sufficient overlap. Will I get a different stitching result if I just stitch the top 2 images rather than stitching all 3 images? I have encountered an issue where depending on the included images, the stitching between the same images can change, which I don't understand.

Thank you!!

![image](https://github.com/user-attachments/assets/eaf67e45-a57e-49af-8162-d8e2a532d35c)

**Comment by lukasalexanderweber** [comment]
maybe https://github.com/OpenStitching/stitching/discussions/135#discussioncomment-6816177 helps you understand the difference between Stitcher and AffineStitcher
